java2typescript-jackson/src/main/java/java2typescript/jackson/module/writer/DeclaredModuleFormatWriter.java

```diff
@@ -0,0 +1,25 @@
+package java2typescript.jackson.module.writer;
+
+import static java.lang.String.format;
+
+import java.io.IOException;
+import java.io.Writer;
+
+import java2typescript.jackson.module.grammar.Module;
+
+/**
+ * Generates TypeScript type definitions for given module in declared module format
+ */
+public class DeclaredModuleFormatWriter extends ExternalModuleFormatWriter {
```

Comment:
Good call. I'll rename to ambient. Found some other relevant issues:

- [typings renaming `ambient` to `global`](https://github.com/typings/typings/issues/343)
- [typescript handbook entry on modules](https://github.com/Microsoft/TypeScript-Handbook/blob/master/pages/Namespaces.md)